### ISSUE TITLE
Inverse Laplace transform of a simple function fails after updating from 0.7.5 to 0.7.6

### ISSUE THREAD

**Issue by jfosorio** [sympy/sympy#8514]
The following transfer function, which appears recurrently in control theory,  produces and error after updating to the new version 0.7.6. 

```python
zeta, wn = symbols('\zeta,\omega_n', positive=True)
t = symbols('t', positive=True)
s = symbols('s')
ft = transforms.inverse_laplace_transform(s/(s**2+2*zeta*wn*s+wn**2),s, t)
```
The error shown is: 

TypeError: cannot determine truth value of
-\omega_n*\zeta - \omega_n*sqrt(\zeta + 1)*cos(atan2(0, \zeta - 1)/2)*sqrt(Abs(\zeta - 1)) < oo

**Comment by jfosorio** [comment]
Hi asmeurer, 
Did you see some new developments regarding this topic?. I read a bit more about the MellinTransform and don't see any reason what this solution is not correct, letting  the function left return True if the integration limits are None and oo. This agrees with the way the function worked in the previous versions. I can continue in this direction but I don't see from your side  guys any interest (so I don't want to loss my time in it if this is the case). This problem is important in many classical control engineering problems but some how it does not seem to catch anybody attention. 
Juan

**Comment by moorepants** [comment]
Juan,

It isn't necessarily that there is no interest. There are likely many interested parties, but they may not have time to delve into this. We certainly would like the bug fixed but we have something on the order of 1600 issues. If you try to find the solution we will do our best to review your pull requests and give you feedback. You can also use git blame to find out who wrote the code and try to ask them directly for help. Thanks for your concern.

**Comment by jfosorio** [comment]
Moorepants, 
Thanks for your answer and the hint.  I also understand the complexity of the task you guys have. I already posted a proposed solution ea510b8. Should I submit a PR with that ?

**Comment by moorepants** [comment]
Yes, please submit a PR and we will review the change.

**Comment by jfosorio** [comment]
Some examples are failing I am trying to see what is the reason.

**Comment by jfosorio** [comment]
The last version is passing both the tests and the doctests  and give similar results that the previous Sympy version. Please review it.

**Comment by jfosorio** [comment]
This is closed by the PR  #9437
